test/e2e_node/memory_eviction_test.go

```diff
@@ -93,11 +99,6 @@ var _ = framework.KubeDescribe("MemoryEvictionSimple [FLAKY] [Slow] [Serial] [Di
 			}, 5*time.Minute, 5*time.Second).Should(BeNil())
 		})
 		It("should drop the node memory pressure condition", func() {
-			By("deleting the memory hog pod.")
```

Comment:
Why not delete the pod? Deletion should be a no-op right even if the pod got evicted?